components/Card.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <UCard class="w-[100%]" @click="handleCardClick(items.id)">
```

Comment:
do we need div here?
you can pass `to="/products/${$props.productId}"` to card and it will convert into link

Reply:
Resolved